TWLight/users/templates/users/collection_tile.html

```diff
@@ -97,14 +81,14 @@
       <hr>
     {% endif %}
     <i class="fa fa-info-circle" aria-hidden="true" title="Partner description page"></i>
-    <strong><a href="{% url 'partners:detail' each_partner.pk %}">{{ each_partner }}</a></strong>
-    {% if each_authorization.stream %}({{ each_authorization.stream }}){% endif %}
-    {% if not each_authorization.is_bundle %}
-      {% if each_authorization.latest_sent_app and not each_authorization.open_app %}
+    <strong><a href="{% url 'partners:detail' collection.partner.pk %}">{{ collection.partner }}</a></strong>
+    {% if collection.stream %}({{ collection.stream }}){% endif %}
+    {% if not collection.authorization.is_bundle %}
+      {% if collection.authorization.latest_sent_app and not collection.authorization.open_app %}
         {% comment %} Translators: Hovertext for when renewals for a partner is not available and the renew button is disabled. {% endcomment %}
-        <span class="pull-right" {% if not each_partner.renewals_available %}title="{% trans 'Renewals are not available for this partner' %}"{% endif %}>
-          <a href="{% url 'applications:renew' each_authorization.latest_sent_app.pk %}" class="btn btn-default pull-right {% if not each_partner.renewals_available %}disabled{% endif %}" role="button">
-            {% if each_authorization.is_valid %}
+        <span class="pull-right" {% if not collection.partner.renewals_available %}title="{% trans 'Renewals are not available for this partner' %}"{% endif %}>
+          <a href="{% url 'applications:renew' collection.authorization.latest_sent_app.pk %}" class="btn btn-default pull-right {% if not collection.partner.renewals_available %}disabled{% endif %}" role="button">
+            {% if collection.authorization.is_valid %}
```

Comment:
I feel like there is room to rethink areas like this, but that's very much a code quality backlog kind of thought, not a launch blocker.